src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_InterpolatedString.vb

```diff
@@ -78,117 +79,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             '
             ' (2) For the built-in types, we can use .ToString(string format) for some format strings.
             '     Detect those cases that can be handled that way and take advantage of them.
```

Comment:
Should we remove this comment now, since it's done in initial binding? Or mention that this transformation was done during initial binding and the result is in `ConstructionOpt`?

Reply:
> Should we remove this comment now, since it's done in initial binding? Or mention that this transformation was done during initial binding and the result is in `ConstructionOpt`?

I do not think there is a need to do either.